pymc3/distributions/continuous.py

```diff
@@ -1381,26 +1409,50 @@ class Exponential(PositiveContinuous):
     Variance  :math:`\dfrac{1}{\lambda^2}`
     ========  ============================
 
+    The exponential distribution can be parametrized either in terms of lam or mu.
+    The link between the parametrizations is given by
+
+    .. math::
+
+        \mu &= \frac{1}{lam}
+
     Parameters
     ----------
     lam: float
         Rate or inverse scale (lam > 0)
+    mu: float
+        Alternative mean (mu > 0)
     """
     rv_op = exponential
 
     @classmethod
-    def dist(cls, lam, *args, **kwargs):
-        lam = aet.as_tensor_variable(floatX(lam))
+    def dist(cls, lam=None, mu=None, *args, **kwargs):
```

Comment:
Introducing new parametrization `mu`. This is not strictly needed, but since we have to reparametrize anway behind the scenes for the `exponential Op`, we might as well add it for the logp.